packages/legend-studio/src/models/protocols/pure/v1/transformation/pureGraph/from/V1_MappingTransformer.ts

```diff
@@ -342,6 +342,7 @@ const transformClassMappingPropertyMappings = (
   values: PropertyMapping[],
   isTransformingEmbeddedPropertyMapping: boolean,
```

Comment:
Looking at all these flags, I start to think we can be a little bit more organized by introducing a new separate `transform context` for mapping transformation, something like:

```
class V1_MappingTransformerContext extends V1_GraphTransformerContext {
  isTransformingEmbeddedPropertyMapping: boolean;
  renderSourceId: boolean;
}
``` 

I'll put a note for this.